moveit_core/collision_detection_fcl/src/collision_robot_fcl.cpp

```diff
@@ -85,7 +85,8 @@ void CollisionRobotFCL::getAttachedBodyObjects(const robot_state::AttachedBody*
   const std::vector<shapes::ShapeConstPtr>& shapes = ab->getShapes();
   for (std::size_t i = 0; i < shapes.size(); ++i)
   {
-    FCLGeometryConstPtr co = createCollisionGeometry(shapes[i], ab, i);
+    FCLGeometryConstPtr co = createCollisionGeometry(shapes[i], getLinkScale(ab->getAttachedLinkName()),
+                                                    getLinkPadding(ab->getAttachedLinkName()), ab, i);
```

Comment:
Needs a space to pass the formatting check:
```suggestion
                                                     getLinkPadding(ab->getAttachedLinkName()), ab, i);
```
(See comments at the top of [`.pre-commit-config.yaml`](https://github.com/ros-planning/moveit/blob/master/.pre-commit-config.yaml))